src/main/java/serverutils/command/pregen/CmdStart.java

```diff
@@ -0,0 +1,80 @@
+package serverutils.command.pregen;
+
+import java.io.IOException;
+import java.util.Objects;
+
+import net.minecraft.command.ICommandSender;
+import net.minecraft.command.WrongUsageException;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.ChatComponentText;
+
+import serverutils.lib.command.CmdBase;
+import serverutils.lib.util.misc.PregeneratorCommandInfo;
+import serverutils.pregenerator.ChunkLoaderManager;
+
+public class CmdStart extends CmdBase {
+
+    public CmdStart() {
+        super("start", Level.OP);
+    }
+
+    @Override
+    public void processCommand(ICommandSender sender, String[] args) {
+        checkArgs(sender, args, 3);
+        if (args.length > 2) {
+            double xLoc, zLoc;
+            if (args[0].equals("~")) {
```

Comment:
Instead of having to use  `~` (which isn't documented anywhere) to start from the player position you could make the x, z optional and use the player position if they aren't provided